gazenot/src/lib.rs

```diff
@@ -0,0 +1,142 @@
+//! Gaze Not Into The Abyss, Lest You Become A Release Engineer.
+//!
+//! Gazenot is a client library for accessing the Abyss service,
+//! which hosts Releases of various Packages (apps).
+//!
+//! Shipping applications with the Abyss is a 5 step process:
+//!
+//! ```no_run
```

Comment:
Looks like the docstring got snipped?

Reply:
main.rs will go here, yes (really annoying to keep doc example in sync)